sdks/go/node/api/urltemplates/urlTemplates.go

```diff
@@ -0,0 +1,11 @@
+package urltemplates
+
+/* resources */
```

Comment:
moving this was required after moving `Listen` out of `cli` to `sdks/go/node/api` to avoid import cycles